docs/docs/concepts/dialogue-understanding.mdx

```diff
@@ -21,9 +25,62 @@ In the CALM approach, _Dialogue Understanding_ aims to represent how the end use
 wants to progress the conversation.
 There is currently only one command generator available: the `LLMCommandGenerator`.
 
-## Using the LLMCommandGenerator in Your Bot
+Like its name indicates, the `CommandGenerator` generates "commands" that are then internally processed to
+trigger operations on the current conversation. Here is a reference of all supported commands:
+
+### StartFlowCommand
```

Comment:
given that when debugging, users will look at the output of the LLM and see `StartFlow()` rather than `StartFlowCommand`, would it be better to document this using the shorter name? IMO the 'command' part at the end is just the class name, not the name of the command itself

Reply:
I've specifically asked about this to Tom [here](https://rasa-hq.slack.com/archives/C05KCQDV8PJ/p1695653745744559).

Reply:
thanks for the context. I don't love the way it looks (repeating Command everywhere) but it's not that important

Reply:
an alternative could be to not make it look like a python object and use spaces, for instance `### Start Flow`. WDYT?

Reply:
yeah that works too

Reply:
b8bb84c469b